_includes/carpentries.html

```diff
@@ -1,5 +1,5 @@
 {% comment %}
-  General description of Software and Data Carpentry.
+  General description of Software, Data, and Library Carpentry.
```

Comment:
Is this a "manual" change or the result of syncing with https://github.com/carpentries/styles? I seem to remember @fmichonneau doing this. I see the advantage would be that we catch all the updates to the boilerplate without missing any.

Reply:
for this file, this edit will be overridden the next time the template will be updated. I don't think it's worth reverting it here but it's good to keep in mind.

Reply:
This is the result of syncing across carpentries.html from https://github.com/carpentries/styles  
This particular file being outdated was causing the most issues with the non-episode content being out of sync.

Reply:
Also, I did note a few things in https://github.com/carpentries/styles that seem to be slightly off or out of date. I'll check issues there and submit an issue if there is not one already.